docs/configurations.md

```diff
@@ -0,0 +1,77 @@
+BALENA_SUPERVISOR_DELTA_REQUEST_TIMEOUT | integer | 30000 | Millisecond time within which an HTTP response containing delta metadata must be returned by the delta server. | v2.9.0
+BALENA_SUPERVISOR_LOCAL_MODE | boolean | false | Enable / Disable local-mode. | v4.0.0
+BALENA_SUPERVISOR_DELTA_APPLY_TIMEOUT | integer | 0 | Millisecond time within which delta data will begin syncing to the local device. | v6.2.0
+BALENA_SUPERVISOR_DELTA_RETRY_COUNT | integer | 30 | Number of retry attempts for downloading deltas. | v6.2.0
+BALENA_SUPERVISOR_DELTA_RETRY_INTERVAL | integer | 10000 | Millisecond time before another delta download is attempted after failiure. | v6.2.0
+BALENA_SUPERVISOR_DELTA_VERSION | integer | 3 | Specifies the delta version to use which has different APIs and download algorithms. | v7.9.0
+BALENA_SUPERVISOR_PERSISTENT_LOGGING | boolean | false | Persists journal logs to disk so they will be available after a reboot event. | v7.15.0
+BALENA_SUPERVISOR_INSTANT_UPDATE_TRIGGER | boolean | true | Enable / Disable instant triggering of updates when a new release is deployed. If set to false, the device will ignore the notification that is triggered when the device's target state has changed. In this case, the device will rely on polling to apply updates. Coupled with a large BALENA_SUPERVISOR_POLL_INTERVAL, this allows spreading out updates in large fleets to avoid overloading local networks when there is a large number of devices at one location. | v9.13.0
+BALENA_HOST_FIREWALL_MODE | string | off | Toggle firewall modes between on, off, and auto. | v11.9.1
+BALENA_HOST_DISCOVERABILITY | boolean | true | Enable / Disable Avahi to run which will allow the device to respond to requests such as network scans. | v11.9.2
+BALENA_HOST_SPLASH_IMAGE | integer | /boot/splash/balena-logo-default.png | Allows changing splash screen on boot to user defined file from userspace. | v12.3.0
+---
+
+In addition to these values, there may be some device-type specific configuration variables that can be set.
+
+## Config.txt
+
+> Supported boards: Any Raspberry Pi and BalenaFin.
+
+This device type utilizes a `config.txt` [file](https://www.raspberrypi.org/documentation/configuration/config-txt/README.md) which we offer some values to be configured.
+
+Name | Type | Default | Description | Valid from
+--- | --- | --- | --- | ---
+BALENA_HOST_CONFIG_disable_splash | integer | 1 | Enable / Disable the Raspberry Pi rainbow splash screen. | v1.0.0
+BALENA_HOST_CONFIG_dtparam | string | "i2c_arm=on","spi=on","audio=on" | Define DT parameters. | v1.0.0
+BALENA_HOST_CONFIG_gpu_mem | integer | 16 | Define device GPU memory in megabytes. | v1.0.0
```

Comment:
What about the following?
- *_HOST_CONFIG_dtoverlay
- *_HOST_CONFIG_device_tree_param
- *_HOST_CONFIG_device_tree_overlay
- *_HOST_CONFIG_gpio
- *_HOST_CONFIG_enable_uart

They're all listed (except `enable_uart`) in config-txt.ts, and `enable_uart` & `dtoverlay` at least are toggle-able options on the dashboard for RPis. Is there a limit to the kinds of config vars listed here?

Reply:
I've added the rest except enable_uart as it is not in the config.txt backend however it is a valid configuration otherwise just not one the Supervisor will apply https://github.com/balena-os/balena-supervisor/blob/2b30ce3b49f7286c74aa86556fad1f731223e783/src/config/backends/config-txt.ts#L32-L38